templates/node/events/index.js

```diff
@@ -13,37 +15,27 @@ const { CloudEvent, HTTP } = require('cloudevents');
  *  -H'Ce-specversion: 1.0' \
  *  http://localhost:8080
  *
- * The event data is extracted from the incoming event and provided as the first
- * parameter to the function. The CloudEvent object itself may be accessed via the
- * context parameter, For example:
- *
- * const incomingEvent = context.cloudevent;
+ * For convenience, the runtime will extract the event from the
+ * CloudEvent envelope and pass it in the second argument to the
+ * function. The entire envelope, along with the HTTP request
+ * metadata, is available from the first argument, 'context'.
  *
  * @param {Context} context the invocation context
- * @param {Object} user the CloudEvent data. If the data content type is application/json
- * this will be converted to an Object via JSON.parse()
+ * @param {Object} event the CloudEvent data
  */
-function verifyUser(context, user) {
-  if (!context.cloudevent) {
-    return {
-      message: 'No cloud event received'
-    };
-  }
+function handle(context, event) {
```

Comment:
I tend to agree with Lance on the naming of this param. I'd prefer `data` or `eventData`.

Reply:
Why?